nextflow.config

```diff
@@ -81,12 +81,18 @@ params {
     save_runmerged_reads        = false
 
     // Host Removal
-    perform_shortread_hostremoval     = false
-    shortread_hostremoval_reference   = null
-    shortread_hostremoval_index       = null
-    save_hostremoval_index            = false
-    save_hostremoval_mapped           = false
-    save_hostremoval_unmapped         = false
+    perform_shortread_hostremoval          = false
+    shortread_hostremoval_reference        = null
+    shortread_hostremoval_index            = null
+    save_hostremoval_index                 = false
+    save_hostremoval_mapped                = false
+    save_hostremoval_unmapped              = false
+    longread_hostremoval_index             = null
+    save_minimap2_hostremoval_index        = false
+    save_minimap2_hostremoval_mapped       = false
+    save_samtools_unmapped_bam             = false
+    save_minimap2_unmapped_fq              = false
```

Comment:
Thanks for the review!

Yes, that's a good idea! So maybe I can drop the `save_samtools_unmapped_bam` parameter? 

It would definitely simplify if we have same parameters for host removal for both short and long reads. For this version I just added separate parameters so that there would be more customisability. I don't have personally so strong opinions on either direction. What do you think? Shall I change it so that host removal is executed for both long and short reads with same parameters?

Reply:
IMO I would assume if I wanted to have those reads specifically, it wouldn't matter if it's short or long read... the purpose to investigate them furhter would be similar I would assume

Reply:
Yes, that makes sense. I'll make the adjustments in the code.